spec/models/post_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(title: 'My First Post', comments_counter: 4, likes_counter: 20) }
+
+  before { subject.save }
+
+  it 'title should be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'comments_counter should be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'likes_counter should be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Kindly fix these two tests `comments_counter should be present` and `likes_counter should be present` as it tests the existence of the title.